### ISSUE TITLE
Set Explicit Width And Height In HTML On All Images?

### ISSUE THREAD

**Issue by alexfinnarn** [CUCentralAdvancement/essential-cu#67]
TLDR; Image dimensions will be provided for all images, but they probably aren't needed. However, image/container dimensions should be placed in HTML and not only CSS...according to the following article which explains in way more detail. 

https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/

@tskillern and @duquennoisa While updating the data layer with how images are now hosted in the CMS, I got to metadata, like width and height, and how that will be used in the IR20 project. These dimensions can be provided per image.

However, the design doesn't have any variable width or height images that I can see. Therefore, images in the codebase can use whatever the container is supposed to be set at for width and height, e.g. `<img src="https://some.site.com/image.jpg" width="200px" height="300px" />` for all card images on the homepage since those are all the same dimensions...probably will use CSS grid here but just an example.

Kelly might have a plan already for how he is going to do this, and I think setting dimensions on a container would avoid layout shifting...but I just wanted to put this issue in here since I'm not sure what the implementation will be. 

CC @mattroush .

**Comment by alexfinnarn** [comment]
assigning to myself to indicate Voltage doesn't need to look at this until CU resources discuss.

**Comment by kmackvolt** [comment]
@alexfinnarn I have been developing everything on this project fluidly responsive, as is normal practice (vs having multiple fixed hard-coded breakpoint changes). So all images in the site are sized fluidly via CSS.
